### ISSUE TITLE
Caddy reverse proxy not working for radarr

### ISSUE THREAD

**Issue by SimpleMachine247** [zilexa/Homeserver#25]
Hi again, I'm sure you're probably tired of hearing from me now so I apologize. 

Issue:
After updating Adguard to dns rewrite radarr.home to my server IP, http://radarr.home/ will not connect and gets immediately updated to https. This does not happen on any of the other *arr apps, just radarr. 

Any idea what would make Radarr function differently from the rest?

**Comment by zilexa** [comment]
No no, you require me to review my own actions. Sometimes it's been a while for me..

But you are 100% correct! I ran into this issue as well. Not sure how I solved it actually. And I also forgot it was an issue for a long time > that's why it's not in the guide, because the  initial reason for documenting my steps was not to forget what I had ran into before..

I don't know why it currently works for me. Could be my browser caching something. 
Consider disabling "certificate validation" in Radarr Settings>General.

I'll try to dig into this tomorrow.

You have gotten very far btw! Congrats!

**Comment by SimpleMachine247** [comment]
Well I went and deleted all browsing data from chrome and now it's working, so you were right on with the browser cache. Must have been stuck since Radarr had ssl and certificate validation on by default when it was loaded the first time. 

Thanks for your help with all these issues. It seems I have everything up and running as expected now. I learned a ton going through your setup!

**Comment by zilexa** [comment]
@SimpleMachine247 do note I made several changes and additions, making the Nightly obsolete, the Monthly now creates nice email with useful info at the top of the email and an important fix for backups on April 16:
Also with 1 command you will receive emails when drives start to fail, this was added in prep-server. 

https://github.com/zilexa/Homeserver/issues/12
Highly recommend going through this list, from the date you executed the prep-server.sh script. 

With all these changes the server runs more unattended now. Also the maintenance scripts and backup scripts really required those updates.
